ipaserver/install/ipa_cacert_manage.py

```diff
@@ -546,6 +543,31 @@ def delete(self):
                 conn.delete_entry(dn)
                 return
 
+    def delete(self):
+        options = self.options
+        nickname = self.args[1]
+        self._delete_by_nickname(nickname, options)
+
+    def prune(self):
+        expired_certs = []
+        ca_certs = certstore.get_ca_certs_nss(api.Backend.ldap2,
+                                              api.env.basedn,
+                                              api.env.realm,
+                                              False)
+
+        now = datetime.datetime.utcnow()
+        for ca_cert, ca_nickname, _ca_trust_flags in ca_certs:
+            if ca_cert.not_valid_after < now:
+                expired_certs.append(ca_nickname)
+                self._delete_by_nickname(ca_nickname, self.options)
+
+        if expired_certs:
+            print("Expired certificates deleted:")
+            for nickname in expired_certs:
+                print(nickname)
+        else:
+            print("No certificates were deleted")
```

Comment:
Should we suggest users run ipa-certupdate on all enrolled machines afterward to apply the removed certs?